Src/DependencyCollector/Shared/HttpCoreDiagnosticSourceListener.cs

```diff
@@ -350,6 +351,8 @@ internal void OnActivityStop(HttpResponseMessage response, HttpRequestMessage re
             if (response != null)
             {
                 this.ParseResponse(response, telemetry);
+                telemetry.OperationDetails[RemoteDependencyConstants.HttpResponseOperationDetailName] = response;
+                telemetry.OperationDetails[RemoteDependencyConstants.HttpResponseHeadersOperationDetailName] = response.Headers;
```

Comment:
why do we need to populate response AND headers? I'd prefer them to be mutually exclusive. 
In case of .NET Core there is no case when a response is missing, but headers are present

Reply:
That was actually how I originally had it (until I hit the corner case where the response was not present).  My rationale for always including them was to make development of a initializer easier -- if the developer is only interested in headers, headers is always populated.  This is not a stance of principal or conviction -- you are much more familiar with what the developer experience of the initializers should be, so I will defer to you.  Please ack that you would prefer to only have the headers when the response is not available and I will make the change after I finish an update on the SDK pull request.